### PR TITLE
[workloads] Update the workload descriptions

### PR DESCRIPTION
**Pull request by lewing**

### CODE REVIEW

**Comment by lewing** [comment]
looks like we need to get the translations properly updated

**Review by steveisok — APPROVED** [review]

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/localize/WorkloadManifest.en.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "workloads/wasm-tools-net6/description": ".NET WebAssembly build tools for net6.0"
+  "workloads/wasm-tools-net6/description": ".NET WebAssembly build tools for net6.0",
+  "workloads/runtimes-windows-net7/description": "Windows Runtime Packs for net6.0"
```
Comment (joeloff) [inline src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/localize/WorkloadManifest.en.json:3]: Should that say .net 7.0 for the one on line 3?

File: src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in [hunk 2]
```diff
@@ -119,7 +119,7 @@
       "platforms": [ "win-x64", "osx-arm64", "osx-x64" ]
```
Comment (joeloff) [inline src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in:119]: Probably unrelated to this, but are we missing arm64 from the platforms here? (It just kind of caught my eye)

Comment (lewing) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in:119]: https://github.com/dotnet/runtime/pull/83162

Comment (lewing) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in:119]: oh wait... good question
@steveisok ?

Comment (steveisok) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in:119]: Yes, that's another one that should be there.

Comment (lewing) [reply src/mono/nuget/Microsoft.NET.Workload.Mono.Toolchain.net6.Manifest/WorkloadManifest.json.in:119]: https://github.com/dotnet/runtime/pull/83237

**Pull request merged**